react-ui/src/components/Reviewer/GroupNames.js

```diff
@@ -45,9 +49,6 @@ class GroupNames extends Component {
     });
   }
 
-  // TODO
-  // Warning: setState(...): Can only update a mounted or mounting component. This usually means you called setState() on an unmounted component. This is a no-op. Please check the code for the GroupNames component.
```

Comment:
This looks like copy/pasta from a console error? Didn't see this happening. If you think it is still a problem could you explain what was happening for you?

Reply:
I think I fixed that problem.